src/test/run-pass/macros/issue-61053.rs

```diff
@@ -0,0 +1,34 @@
+// run-pass
```

Comment:
Could you rename the tests from `issue-61053` into something meaningful, like `meta-variable-misuse`.